pkg/kubelet/cm/cpumanager/state/state_mem.go

```diff
@@ -93,7 +93,7 @@ func (s *stateMemory) SetCPUAssignments(a ContainerCPUAssignments) {
 	defer s.Unlock()
 
 	s.assignments = a.Clone()
-	klog.InfoS("Updated CPUSet assignments", "assignments", a)
+	klog.Infof("Updated CPUSet assignments. assignments=%v", a)
```

Comment:
while I seen the behavior myself, I'd rather fix `InfoS` somehow than drop it, especially because we want to move towards structured logging

Reply:
Reverted back to `InfoS` and added a `String()` function instead